crates/consensus/common/src/validation.rs

```diff
@@ -38,6 +43,33 @@ pub fn validate_header_standalone(
         return Err(ConsensusError::WithdrawalsRootUnexpected)
     }
 
+    // Ensures that EIP-4844 fields exist once cancun is active, and have the following properties:
+    //
+    //  * blob_gas_used is less than or equal to MAX_DATA_GAS_PER_BLOCK
+    //  * blob_gas_used is a multiple of DATA_GAS_PER_BLOB
+    //
+    if chain_spec.fork(Hardfork::Cancun).active_at_timestamp(header.timestamp) {
+        let blob_gas_used = header.blob_gas_used.ok_or(ConsensusError::BlobGasUsedMissing)?;
```

Comment:
just realized our full block client _enforces_ that the header has a withdrawals root, which is not true for pre-shanghai. but we can only perform these checks properly if we have the `Arc<ChainSpec>`, so it will need to be added

Reply:
so yes they need to be updated

Reply:
^ nevermind, I misread the code and this is completely wrong. But we do need to update the checks